ReadMe.md

```diff
@@ -157,11 +160,18 @@ Or for the Pro version with additional features:
    ```csharp
    builder.Services.AddGeoBlazor(builder.Configuration);
    ```
+8. GeoBlazor requires Interactive server-side rendering enabled. Verify that the following lines are present in your `Program.cs`:
+
+   ```csharp
+      .AddInteractiveServerComponents(); // this should be in the Services section
 
+      .AddInteractiveServerRenderMode(); // this should be in the app section
+   ```
 8. Create a Razor Component page with a map:
 
    ```html
    @page "/"
+   @rendermode InteractiveServer
```

Comment:
No great way to show an inline comment here, so might put the other options into the text above or below, like:

8. Create a Razor Component page with a map. Be sure to set an [Interactive Render Mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes) like Server, WebAssembly, or Auto: